src/main/java/com/microsoft/azure/spring/data/documentdb/repository/DocumentDbRepository.java

```diff
@@ -6,8 +6,8 @@
 
 package com.microsoft.azure.spring.data.documentdb.repository;
 
-import org.springframework.data.repository.NoRepositoryBean;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.NoRepositoryBean;
```

Comment:
remove

Reply:
This is necessary, I used intellij optimized the import, so the import order changed a little bit.